nupd/fetchers/github.py

```diff
@@ -33,18 +47,63 @@ class GHRepository:
     owner: str
     repo: str
     branch: str
-
+    meta: MetaInformation = field(
+        converter=lambda x: MetaInformation(**x)
+        if not isinstance(x, MetaInformation)
+        else x
+    )
     has_submodules: bool | None
     commit: Commit | None = field(
         converter=lambda x: Commit(**x)
         if not isinstance(x, Commit | None)
         else x
     )
-    meta: MetaInformation = field(
-        converter=lambda x: MetaInformation(**x)
-        if not isinstance(x, MetaInformation)
-        else x
-    )
+    latest_release: GitHubRelease | None = None
```

Comment:
This shouldn't be optional if we fetch latest release by default

Reply:
Not every repository has a release.

Reply:
Oh, you are right. Then
```suggestion
    latest_release: GitHubRelease | None
```

Reply:
That makes it a required argument

```bash
ERROR tests/fetchers/github/test_other.py::test_prefetch_commit_on_repo_class - TypeError: GHRepository.__init__() missing 1 required positional argument: 'latest_release'
ERROR tests/fetchers/github/test_other.py::test_github_prefetch_commit - TypeError: GHRepository.__init__() missing 1 required positional argument: 'latest_release'
ERROR tests/fetchers/github/test_other.py::test_prefetch_commit_already_fetched - TypeError: GHRepository.__init__() missing 1 required positional argument: 'latest_release'
ERROR tests/fetchers/github/test_other.py::test_prefetch_commit_fail - TypeError: GHRepository.__init__() missing 1 required positional argument: 'latest_release'
ERROR tests/fetchers/github/test_other.py::test_get_prefetch_url - TypeError: GHRepository.__init__() missing 1 required positional argument: 'latest_release'
ERROR tests/fetchers/github/test_other.py::test_get_prefetch_url_no_commit - TypeError: GHRepository.__init__() missing 1 required positional argument: 'latest_release'
ERROR tests/fetchers/github/test_other.py::test_github_does_have_submodules - TypeError: GHRepository.__init__() missing 1 required positional argument: 'latest_release'
ERROR tests/fetchers/github/test_other.py::test_github_does_have_submodules_not_found - TypeError: GHRepository.__init__() missing 1 required positional argument: 'latest_release'
ERROR tests/fetchers/github/test_other.py::test_github_does_have_submodules_already_fetched - TypeError: GHRepository.__init__() missing 1 required positional argument: 'latest_release'

Reply:
Updated, since it looks like that's what you're doing with commit, has_submodules etc